Source/Core/SMTLibOptions.cs

```diff
@@ -20,5 +20,13 @@ public interface SMTLibOptions
     bool SIBoolControlVC { get; }
     bool RestartProverPerVC { get; }
     bool Trace { get; }
+    
+    /**
+     * Setting this to true will rename all identifiers in the Boogie program to a generated name that does not depend on the original name.
```

Comment:
"all identifiers" refers to both global and local names, right?

Reply:
Jep